bob/src/api/mod.rs

```diff
@@ -145,12 +145,18 @@ pub(crate) struct NodeConfiguration {
 }
 
 #[derive(Debug, Serialize)]
-pub(crate) struct SpaceInfo {
+struct Space {
     total_disk_space_bytes: u64,
     free_disk_space_bytes: u64,
     used_disk_space_bytes: u64,
     occupied_disk_space_bytes: u64,
```

Comment:
If we decide to allow breaking changes, then this field should be renamed to `occupied_by_bob_disk_space_bytes`. Such a name will better reflect the meaning of the field